### DISCUSSION TITLE
Resigned app crash on launch

### DISCUSSION THREAD

**Discussion by tburakdemir** [fastlane/fastlane#21920] [Q&A]
Hello,

I want to resign an app but it crashes on launch.

Original ipa form app store:
https://drive.google.com/file/d/1biZV7jdjTy2iTf32jXjr35hCsx6EulOw/view?usp=sharing

Copy it
cp cp com.netflix.Netflix_363590051_16.22.0.ipa netflix2.ipa   

The ipa i resigned with this command:
fastlane run resign ipa:"netflix2.ipa" signing_identity:"Apple Development: Device Manager (CZ68MXXXXX)" provisioning_profile:"dotcomnew210224.mobileprovision"

`[13:24:05]: fastlane detected a Gemfile in the current directory
[13:24:05]: However, it seems like you didn't use `bundle exec`
[13:24:05]: To launch fastlane faster, please use
[13:24:05]: 
[13:24:05]: $ bundle exec fastlane run resign ipa:netflix2.ipa signing_identity:Apple Development: Device Manager (CZ68MXN472) provisioning_profile:dotcomnew210224.mobileprovision
[13:24:05]: 
[13:24:05]: Get started using a Gemfile for fastlane https://docs.fastlane.tools/getting-started/ios/setup/#use-a-gemfile
[13:24:06]: --------------------
[13:24:06]: --- Step: resign ---
[13:24:06]: --------------------
/opt/homebrew/Cellar/fastlane/2.219.0_2/libexec/gems/fastlane-2.219.0/sigh/lib/assets/resign.sh netflix2.ipa C33561A65D2BD4A04DDAC96B53AD777810E21DA3 -p /Users/burak/Projects/ipas/dotcomnew210224.mobileprovision          netflix2.ipa
cp: _floatsignTemp/Payload/Argo.app/PlugIns/pushImageService.appex/embedded.mobileprovision: No such file or directory
_floatsignTemp/Payload/Argo.app/PlugIns/pushImageService.appex: replacing existing signature
cp: _floatsignTemp/Payload/Argo.app/embedded.mobileprovision: No such file or directory
_floatsignTemp/Payload/Argo.app/Frameworks/AppLoggingSystem.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/EventSinkClient.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/FTLProbeClient.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/Falcor.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/MinimalDependencies.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/MslClient.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFCW444.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFContentKey.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFLicenseFetch.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFManifestCommon.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFNetworkState.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFPlatformCommon.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFPlaylistGeneration.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFPrefetchCache.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFRCW.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFSharedStore.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFSqlStore.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFStoreCrypto.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFUIPlayerFoundation.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFURLSession.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/NFWebCrypto.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/Nbp.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/PlayapiClient.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app/Frameworks/ZPClient_-CE73F4E2532E3EF_PackageProduct.framework: replacing existing signature
_floatsignTemp/Payload/Argo.app: replacing existing signature

[13:24:13]: Successfully signed netflix2.ipa!
[13:24:13]: Successfully re-signed .ipa 🔏.
[13:24:13]: Result: true
`

Resigned ipa:
https://drive.google.com/file/d/1fJcQMBW8Am0DK8d6Gy0-fyHo8HDJHlXf/view?usp=drive_link

Install with ideviceinstaller:
ideviceinstaller -i netflix2.ipa
It doesnt give an error. It is installed.

`Copying 'netflix2.ipa' to device... DONE.
Installing 'com.netflix.Netflix'
Install: CreatingStagingDirectory (5%)
Install: ExtractingPackage (15%)
Install: InspectingPackage (20%)
Install: PreflightingApplication (30%)
Install: VerifyingApplication (40%)
Install: CreatingContainer (50%)
Install: InstallingApplication (60%)
Install: PostflightingApplication (70%)
Install: SandboxingApplication (80%)
Install: GeneratingApplicationMap (90%)
Install: InstallComplete (100%)
Install: Complete`

But when i click the app on my phone it crashes on launch.
If i install original one there is no error like this.
Btw in mobileprovisionfile there is my device's udid in it.

Any idea what is wrong?

**Comment by tburakdemir** [comment]
Is there any idea?

**Reply by cergfix** [reply]
What's the old/new bundle ids that you're using? If you check the resigned build entitlements (`unzip netflix2.ipa ; cd Payload/Argo.app ; codesign -d --entitlements - Argo`) then the application-identifier is `[String] 6Q34V79NW3.com.*` which might be the issue.
